lib/utils/promise-utils.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Created by gadyaari on 07/02/2016.
  */
-var logger = require('./../logger/logger')(module);
+var logger = require('../../common/logger').getLogger('promise-utils');
```

Comment:
do you use this logger?, if yes, then you have duplicated name with SynchornizedPromises